db/migrate/20170531194154_add_multiple_redirect_uris.rb

```diff
@@ -0,0 +1,15 @@
+class AddMultipleRedirectUris < ActiveRecord::Migration
+  def up
+    add_column :service_providers, :redirect_uris, :json
```

Comment:
I think it's acceptable as is, how strongly do you feel that it should be changed to array?